action.yml

```
@@ -6,6 +6,10 @@ inputs:
     description: 'Specific dprint version to use (ex. 0.30.3)'
     required: false
     default: ''
+  dprint-config-path:
+    description: 'Specific dprint config to use (ex. dprint.json)'
+    required: true
+    default: 'dprint.json'
```

```suggestion
    default: ''
```

People might use dprint.jsonc